### PR TITLE
update autocomplete and ycmd

### PR DESCRIPTION
**Pull request by pittcat**

### CODE REVIEW

**Comment by liuchengxu** [comment]
I have fixed some minor issues in the last PR. See https://github.com/liuchengxu/space-vim/commit/b08fe6124a7a9a3e49a246cec2fb72f5f232e496. Please take a look.

FYI, you don't have to re-fork the repo every time, you can merge the upstream update in your fork repo.

**Comment by pittcat** [comment]
Thanks for your guide ,and I'm very sorry for pool git skill.

File: layers/+completion/auto-completion/config.vim [hunk 1]
```diff
@@ -1,10 +1,6 @@
 scriptencoding utf-8
 
-if g:spacevim_vim8 || g:spacevim_nvim
-  inoremap <expr> <CR> (pumvisible() ? "\<c-y>\<cr>" : "\<CR>")
-  inoremap <expr> <Tab> pumvisible() ? "\<C-n>" : "\<Tab>"
-  inoremap <expr> <S-Tab> pumvisible() ? "\<C-p>" : "\<S-Tab>"
-else
+if !empty(glob("$HOME/.local/share/nvim/plugged/vim-mucomplete")) ||  !empty(glob("$HOME/.vim/plugged/vim-mucomplete"))
```
Comment (liuchengxu) [inline layers/+completion/auto-completion/config.vim]: I think it's better to use the exposed variable `g:plugs`.
```
if has_key(g:plugs, 'vim-mucomplete')
```

**Comment by liuchengxu** [comment]
Merged, thanks!

**Pull request merged**